src/Storages/ObjectStorage/DataLakes/Iceberg/ManifestFile.cpp

```diff
@@ -265,19 +296,43 @@ ManifestFileContent::ManifestFileContent(
             /// name generation for such subfields (we support names of nested subfields in structs only).
             if (!field_characteristics)
             {
+                LOG_DEBUG(
+                    &Poco::Logger::get("ManifestFileContent::ManifestFileContent"), "Column with id {}, so bounds are ignored", column_id);
                 continue;
             }
             const auto & name_and_type = *field_characteristics;
             String left_str;
             String right_str;
             /// lower_bound and upper_bound may be NULL.
             if (!bounds.first.tryGet(left_str) || !bounds.second.tryGet(right_str))
+            {
+                LOG_DEBUG(
```

Comment:
Do we really need this LOG_DEBUG (and the same below)? It doesn't seem to be so useful